src/cirrus/cirrus_setup.py

```diff
@@ -67,13 +67,15 @@ def create_github_token():
     requests for things like pull requests
 
     """
+    authz_url = urljoin(get_github_api_base(), '/authorizations')
```

Comment:
The `/` is needed in `'authorizations'`?

Reply:
Yes, see the original value (I screwed this up) https://github.com/cloudant/cirrus/pull/41/commits/f4ff175dd8285623b352adf3d03ba7a22cba848c#diff-5a56a3b8782b6cde3e30a41d7669cc59L17